sdk/keychain/keychain.go

```diff
@@ -49,6 +50,20 @@ type Ledger struct {
 	LedgerIndices []uint32
 }
 
+func PrivateKeyToAvalancheAccount(
```

Comment:
why do we want this? Why don't we just reuse https://github.com/ava-labs/avalanche-tooling-sdk-go/blob/fef92d1127dc37da50a9307a78663b81a04b971e/wallet/p-chain/examples/convertSubnetToL1Tx.go#L51?

It handles ledger too

Reply:
Now that i think about it, we should create wallet object directly from private key / private key path, no need for intermediary keychain / we can abstract it out

Reply:
I am trying to align to SDK. they have this account abstraction, very similar to the keychain. Also the function
is there. I believe it is handy. Anyway, not opposite to also having it in wallet. But I would have separate function
for the options:
- ledger
- path
- string